eva/storage/video_storage_engine.py

```diff
@@ -58,3 +58,15 @@ def read(
                     batch.frames[table.columns[0].name] = row_id
                     batch.frames[table.columns[1].name] = str(video_file_name)
                     yield batch
+
+    def clear(self, table: TableCatalogEntry, video_paths: List[Path]):
+        try:
+            media_metadata_table = self._get_metadata_table(table)
+            for video_path in video_paths:
+                dst_file_name = self._xform_file_url_to_file_name(Path(video_path))
+                image_file = Path(table.file_url) / dst_file_name
+                self._rdb_handler.clear(media_metadata_table)
+                image_file.unlink()
```

Comment:
1. I think the variable name should be `video_file` instead of `image_file`
2. Similar to image, this removes the original video.

Reply:
1. Agree. will change in the next commit.
2. It doesn't remove the original video. It removes the video from the table but not the video on the user's drive.